Testscripts/Linux/SetupRDMA.sh

```diff
@@ -606,8 +606,23 @@ function Main() {
 		Verify_File $benchmark_bin
 	fi
 
+    HOMEDIR=$(pwd)
+	cd ~
+	LogMsg "Proceeding OSU MPI Benchmark test installation"
+	LogMsg "Downloading mpi-benchmarks from $osu_mpi_benchmark"
+	wget http://mvapich.cse.ohio-state.edu/download/mvapich/osu-micro-benchmarks-5.6.2.tar.gz
+	tar xvzf osu-micro-benchmarks-5.6.2.tar.gz
```

Comment:
Get the file name from the path in 614 and 615. Refer to other codes inside

Reply:
Done